integration/consul_catalog_test.go

```diff
@@ -62,23 +61,13 @@ func (s *ConsulCatalogSuite) TearDownSuite(c *check.C) {
 	}
 }
 
-func (s *ConsulCatalogSuite) registerService(id, name, address, port string, tags []string, onAgent bool) error {
-	iPort, err := strconv.Atoi(port)
-	if err != nil {
-		return err
-	}
+func (s *ConsulCatalogSuite) registerService(foo *api.AgentServiceRegistration, onAgent bool) error {
```

Comment:
let's use something more sensible than "foo" please.

Reply:
Ah, I hadn't seen we're in a test here too. Would be nice, but not blocking for review then.